types/vector/nested/README.md

```diff
@@ -0,0 +1,13 @@
+# `std::vector<std::vector<std::int32_t>>`
+
+## Fields
+
+ * `[Split]Index{32,64}`
+
+with the corresponding column type for the two collection parent fields.
+
+## Entries
+
+1. One element vectors, with ascending values
+2. Empty vectors
+3. Increasing number of elements in the outer vector
```

Comment:
Perhaps also specify that the inner vectors have arbitrary length, including empty.